scripts/npm-safe-publish.sh

```diff
@@ -67,6 +75,11 @@ if [[ "$NPM_SHOW" != "$NPM_LS" ]]; then
     fi
 
     npm publish --otp=$OTP
+
+    curl -X POST \
```

Comment:
You don't check whether or not the npm publish step succeeded; or whether it actually published.
use `$?` to get the previous return code (in this case, npm)